src/audio3d/Audio3DBuffer.h

```diff
@@ -0,0 +1,113 @@
+#ifndef __PIROGRONIAN__AUDIO_3D__BUFFER__H__ 
```

Comment:
This looks like vanity.

Reply:
It started as separate project and I forgot to change it (or though it's just neglectable). So, changing now :-)

Reply:
I prefer `#pragma once`, that's nonstandard extension but supported by all major compilers. for ages.